input/postgres/relation_stats.go

```diff
@@ -156,3 +169,52 @@ func GetIndexStats(db *sql.DB, postgresVersion state.PostgresVersion, ignoreRege
 
 	return
 }
+
+func GetColumnStats(logger *util.Logger, db *sql.DB, globalCollectionOpts state.CollectionOpts, systemType string) (columnStats state.PostgresColumnStatsMap, err error) {
+	var sourceTable string
+
+	helperExists := false
+	db.QueryRow(QueryMarkerSQL + columnStatsHelperSQL).Scan(&helperExists)
+
+	if helperExists {
+		logger.PrintVerbose("Found pganalyze.get_column_stats() stats helper")
+		sourceTable = "pganalyze.get_column_stats()"
+	} else {
+		if systemType != "heroku" && !connectedAsSuperUser(db, systemType) && !connectedAsMonitoringRole(db) && globalCollectionOpts.TestRun {
```

Comment:
I think we can need to drop the `connectedAsMonitoringRole` part of this condition, since `pg_monitor` does not grant access to `pg_stats` (i.e. the user does need to create the helper always, even on newer Postgres versions)